vmware_exporter/helpers.py

```diff
@@ -4,7 +4,14 @@
 
 
 def get_bool_env(key, default=None):
-    return bool(os.environ.get(key, default))
+    value = os.environ.get(key, default)
+    if isinstance(value, str):
```

Comment:
Whats about:

```
if value is None:
  return default
else:
  return value.lower() == 'true'
```

Reply:
@kremers - it gets complicated, because get_bool_env is passed values of type `bool` for the parameter`default` in some sections of the code. E.g. this snippet under VMWareMetricsResource.configure
```
'ignore_ssl': get_bool_env('VSPHERE_IGNORE_SSL', False),
                'collect_only': {
                    'vms': get_bool_env('VSPHERE_COLLECT_VMS', True),
                    'vmguests': get_bool_env('VSPHERE_COLLECT_VMGUESTS', True),
                    'datastores': get_bool_env('VSPHERE_COLLECT_DATASTORES', True),
                    'hosts': get_bool_env('VSPHERE_COLLECT_HOSTS', True),
                    'snapshots': get_bool_env('VSPHERE_COLLECT_SNAPSHOTS', True),
```
so we end up having to deal with str, bool or None. therefore the check for `isinstance(value, str)`

Reply:
```
def get_bool_env(key: str, default: bool):
  value = os.environ.get(key, default)
  return value if type(value) == bool else value.lower() == 'true'
```
or longer syntax
```
def get_bool_env(key: str, default: bool):
  value = os.environ.get(key, default)
  if type(value) == bool:
    return value
  else:
    return value.lower() == 'true'
```


Does handle all cases? 'None' could never appear since all calls to get_bool_env, get parametrized with bool values, so I would vote for removing it as default value. Not sure if type(....) == bool is more neat than isinstance(value, bool).